xapian-applications/omega/daflist.h

```diff
@@ -0,0 +1,8 @@
+/** @file daflist.h
+ * @brief 'Extensions' of directories to be indexed as files.
+ */
+
+#include <string>
+#include <vector>
+
+std::vector<std::string> dasfile {".key", ".numbers", ".pages"};
```

Comment:
You're still hard-coding a list of extensions in the code.  As I said before this should be configurable in a similar way to how indexing of files is configured.  The user can then configure their own custom filters for these special directories.

For example, to handle `.xcodeproj` directories on macOS they could write a script `xcodeproj2txt` which takes a path to such a directory and extracts text from some files inside, and then run omindex with options `--mime-type=xcodeproj/:application/x-apple-xcode-project --filter=application/x-apple-xcode-project:xcodeproj2txt`